tests/Stellar/TransactionTests.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "../interface/TWTestUtilities.h"
+
+#include "Stellar/Address.h"
+#include "Stellar/Signer.h"
+#include "Bitcoin/Address.h"
+#include "HexCoding.h"
+#include "PrivateKey.h"
+#include <TrustWalletCore/TWHDWallet.h>
+
+#include <gtest/gtest.h>
+
+using namespace std;
+using namespace TW;
+using namespace TW::Stellar;
+
+TEST(StellarTransaction, sign) {
+    auto words = STRING("indicate rival expand cave giant same grocery burden ugly rose tuna blood");
+    auto passphrase = STRING("");
+
+    auto wallet = WRAP(TWHDWallet, TWHDWalletCreateWithMnemonic(words.get(), passphrase.get()));
```

Comment:
@madcake might be better to use just a private key, instead of mnemonic and have separate test for deriving the private key from mnemonic! 

Looks great